requirements.txt

```diff
@@ -1,2 +1,2 @@
-django
+django==4.1
```

Comment:
This would prevent the install of a newer version. I typically would set the minimum version and then go up – for example `django>=4.1` or, if we wanted to restrict it to the ones we had in mind while testing, perhaps `django>=4.1,<6`.

Reply:
Thanks, I will switch to that.

I think I set it as a shortcut for testing with tox. I was trying to figure out how to do a matrix build with tox to test multiple versions of django but I'm not that familiar with tox and I use github actions for matrix builds

Reply:
Yeah, I'd like to pivot to GitHub actions for this as well since it'd be faster but I no longer work on the project where I primarily used this code so it's somewhat far down the to-do list.

Reply:
if I have energy to make a quick stab at github actions based on my other projects would it be ok to include in this pr?